webview_panels/src/modules/documentationEditor/components/tests/forms/TestForm.tsx

```diff
@@ -16,6 +16,8 @@ import Relationships from "./Relationships";
 import { useEffect, useMemo } from "react";
 import useTestFormSave, { TestOperation } from "../hooks/useTestFormSave";
 import { SaveRequest } from "../types";
+import { sendTelemetryEvent } from "../../telemetry";
+import { TelemetryEvents } from "../../../../../../../src/telemetry/events";
```

Comment:
@saravmajestic is this common to provide such relative paths?

Reply:
Actually not. We should set alias and use it. Let me fix this

Reply:
done